x-pack/performance/journeys/cloud_security_dashboard.ts

```diff
@@ -22,7 +22,7 @@ export const journey = new Journey({
     });
   },
   ftrConfigPath: 'x-pack/performance/configs/cloud_security_posture_config.ts',
-  esArchives: ['x-pack/performance/es_archives/kspm_findings'],
+  esArchives: ['x-pack/performance/es_archives/cloud_security_posture_findings'],
```

Comment:
Hi Lola, as we discussed we agreed there is no issue with the test and we found the core issue to be in the given change.

lets revert that and keep the journey test as it was

Reply:
Hi Kfir, I deleted the old `es_archive` data.json  to include both kspm and cspm findings.